docker/Dockerfile

```diff
@@ -43,11 +45,10 @@ RUN git clone -q https://github.com/kthohr/stats \
 # install ACE
 WORKDIR /opt
 
-ADD https://api.github.com/repos/SystemsGenetics/ACE/git/refs/heads/${ACE_REVISION} ace-version.json
 RUN git clone -q https://github.com/SystemsGenetics/ACE.git \
 	&& cd ACE/build \
 	&& git checkout -q ${ACE_REVISION} \
-	&& qmake ../src/ACE.pro GUI=no \
```

Comment:
Why was this setting removed? I believe this setting helped decrease the size of the docker image. I don't think you can use the GUI through a docker image anyway.

Reply:
Probably just my mistake. I can add it back in.

Reply:
Fixed in latest push to the branch